tests/utils.go

```diff
@@ -4918,3 +4930,7 @@ func DetectLatestUpstreamOfficialTag() (string, error) {
 	By(fmt.Sprintf("By detecting latest upstream official tag %s for current branch", tag))
 	return tag, nil
 }
+
+func Retcode(retcode string) string {
```

Comment:
Why is this an exposed function now?

Reply:
in case we would need to use the same logic outside of utils.go (some tests have echo $?)

Reply:
I was very close to nack this PR only because you touched this huge endless file.
Please do not make me regret this overlook.

No one else is using it now, if someone will want to use it the can change it then.